.github/workflows/coding-standards.yml

```diff
@@ -38,3 +38,6 @@ jobs:
       # https://github.com/doctrine/.github/issues/3
       - name: "Run PHP_CodeSniffer"
         run: "vendor/bin/phpcs -q --no-colors --report=checkstyle | cs2pr"
+
+      - name: "Run psalm"
+        run: "composer static-analysis"
```

Comment:
Please introduce a new workflow based on https://github.com/doctrine/.github/blob/main/workflow-templates/static-analysis.yml where psalm will run instead of putting it into the coding style workflow.